tests/integration/questionnaire/test_questionnaire_calculated_summary.py

```diff
@@ -48,31 +48,27 @@ def _complete_calculated_summary_path_no_skip(self):
     def test_calculated_summary(self):
         self.launchSurvey("test_calculated_summary")
         self._complete_calculated_summary_path_with_skip()
-        self.assertInBody("Skipped Fourth")
         self.assertInBody(
             "We calculate the total of currency values entered to be £80.00"
         )
 
     def test_calculated_summary_no_skip(self):
         self.launchSurvey("test_calculated_summary")
         self._complete_calculated_summary_path_no_skip()
-        self.assertNotInBody("Skipped Fourth")
```

Comment:
Instead of removing this shouldn't we check that `"Fourth answer label"` is in body now?

Reply:
Not sure we necessarily need this as it is really just testing the total which is only correct if the fourth is skipped?